geometric/tests/addons.py

```diff
@@ -94,8 +114,9 @@ def workqueue_found():
 def in_folder(request):
 
     # Build out a test folder
+    # * request.node allows separate dirs for parametrized tests in parallel
     cwd = os.path.abspath(os.getcwd())
-    test_folder = os.path.join(cwd, 'test_generated_files', request.function.__name__)
```

Comment:
I assume this is a better way to get the name of the test folder?  Please confirm.

Reply:
Yes, `function.__name__` gets the name of the test function (e.g., `test_lif_bsse`) but `node.name` gets the per-test name that includes parametrization (e.g., `test_lif_bsse[psi4-nocp]`). The previous function way was no trouble when the tests were run in serial, but it led to file overwriting when running pytest in parallel.